x-pack/plugins/lens/public/pie_visualization/render_function.tsx

```diff
@@ -272,8 +286,9 @@ export function PieComponent(
       className="lnsPieExpression__container"
       isReady={state.isReady}
     >
-      <Chart>
+      <Chart ref={chartRef}>
         <Settings
+          onPointerUpdate={handleCursorUpdate}
```

Comment:
@markov00 @nickofthyme this code was added in the hopes that cursor support will one day work for this type of charts as well. If you do not plan to do this in the near future, it is probably worth removing this code. Answer, please

Reply:
If it is provided by the charts team, we want to add it for the `vis_type_pie` too so maybe it worths it to do it then

Reply:
one way or another I'll have to do something about these changes =) Ok, let's wait for the response from the charts team, I vote for the second option (add it for the vis_type_pie too)

Reply:
Yes, I vote for removing it for heatmap and pie as it is not supported yet

Reply:
ok, `cursor` was removed from `heatmap` and `chart`.  When charts add this feature we easily can add it back

Reply:
Great, thanx Alex!